### DISCUSSION TITLE
Amplitude/Output scaling during inference.

### DISCUSSION THREAD

**Discussion by lirus7** [coqui-ai/TTS#2747] [General Q&A]
### Describe the bug

I have trained a VITS model on a custom english speaker. I am facing a problem with the inference of this model. I am able to get the speaker style along with the duration accurately but the amplitude seems to be way-off compared to the ground truth inputs. [I tested this by generating the ground truth wav from the model].

My suspicion is that there is some output scaling happening in the process which is not followed during inference. How do i fix this?

[Generated] https://github.com/coqui-ai/TTS/assets/13689389/9fe5883d-139c-45ad-8de3-26753ff54cd2
[GT] https://github.com/coqui-ai/TTS/assets/13689389/6a43aad1-eca6-44fb-ac2c-26700b5bf32e

### To Reproduce

The dataset consists of audio files of a speaker in english. I have already shared a reference audio.

### Environment

```shell
{
    "CUDA": {
        "GPU": [
            "Tesla T4"
        ],
        "available": true,
        "version": "11.7"
    },
    "Packages": {
        "PyTorch_debug": false,
        "PyTorch_version": "1.13.1+cu117",
        "TTS": "0.11.1",
        "numpy": "1.21.6"
    },
    "System": {
        "OS": "Linux",
        "architecture": [
            "64bit",
            "ELF"
        ],
        "processor": "x86_64",
        "python": "3.9.0",
        "version": "#66~18.04.1-Ubuntu SMP Thu Jan 13 19:45:37 UTC 2022"
    }
}
```

**Comment by erogol** [comment]
I convert it to a discussion since it is not dev related.
